ckan/templates/ckan-deployment.yaml

```diff
@@ -104,6 +104,15 @@ spec:
           mountPath: /etc/ckan-conf/templates
           readOnly: true
       {{ end }}
+      {{ if .Values.ckanPrimaryColor }}
+      - name: themer
+        image: {{ .Values.themerImage | quote }}
+        env:
+        - name: CKAN_PRIMARY_COLOR
```

Comment:
@OriHoch see this commit:

https://github.com/ViderumGlobal/ckan-cloud-docker/commit/e04c6a8c6851f62164003a565826226a2ec29bb1